Cargo.toml

```diff
@@ -14,23 +14,27 @@ readme = "README.md"
 
 [features]
 default = []
-std = ["serde", "proptest", "schemars", "proptest-derive"]
+std = ["serde", "proptest", "schemars", "proptest-derive", "field-offset"]
```

Comment:
Could we hide the field-offset dependency behind a distinct feature flag, please?  IIRC, that crate is no-std friendly and the capabilities related to having it are distinct from the use cases enabled by the other things currently behind the "std" flag.